ingestion/src/metadata/ingestion/source/azure_sql.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""AZURE SQL source module"""
+
+from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
+from metadata.ingestion.source.sql_source import SQLSource
+from metadata.ingestion.source.sql_source_common import SQLConnectionConfig
+
+
+class AzuresqlConfig(SQLConnectionConfig):
+    """AZURE SQL config -- extends SQLConnectionConfig class"""
+
+    host_port = "localhost:1433"
+    scheme = "mssql+pyodbc"
+    service_type = "AZURE SQL"
+    use_pyodbc: bool = True
+    uri_string: str = ""
+
+    def get_connection_url(self):
+        if self.use_pyodbc:
+            self.scheme = "mssql+pyodbc"
+            return f"{self.scheme}://{self.uri_string}"
```

Comment:
We are using pyodbc as default, we won't need a check here in get_connection_url